snuba/datasets/storages/events_common.py

```diff
@@ -176,6 +176,10 @@
         ("culprit", Nullable(String())),
         ("sdk_integrations", Array(String())),
         ("modules", Nested([("name", String()), ("version", String())])),
+        # Aliases
```

Comment:
I don't think there is the need to call out which columns are not present as concrete column in the table schema.

Reply:
I'm confused, isn't that what you asked me to add in the previous comment?

Reply:
I am just talking about the comment. You can remove the comment

Reply:
Ok will do